### ISSUE TITLE
Issue with Bitcode on TestFlight builds

### ISSUE THREAD

**Issue by sho-matsu** [shakebugs/shake-ios#38]
When building with Xcode 16 on Bitrise, the following error occurred, causing the upload to TestFlight to fail.

```
[!] Error uploading ipa file: [Application Loader Error Output]: [ContentDelivery.Uploader.6000004081C0] Asset validation failed (90482) Invalid Executable. The executable ‘cocoreview.app/Frameworks/Shake.framework/Shake’ contains bitcode. (ID: XXXX) [Application Loader Error Output]: Error uploading ‘/var/folders/b8/9zt_tx3s1y37nzh_cs991xyc0000gn/T/74e73391-8bf2-436f-99ea-3670afa2ee10.ipa’. (exit code: 1)
```

This issue was resolved by removing the bitcode from the Shake Framework.

Since we are using CocoaPods, we added the following workaround to the Podfile.

```ruby
post_install do |installer|
  installer.pods_project.targets.each do |target|
    if target.name == 'Shake'
      `xcrun -sdk iphoneos bitcode_strip -r Pods/Shake/Sources/Shake.xcframework/ios-arm64/Shake.framework/Shake -o Pods/Shake/Sources/Shake.xcframework/ios-arm64/Shake.framework/Shake`
    end
  end
end
```

Using Shake 16.2.5 Installed using CocoaPods
Xcode Version: 16.0

**Comment by gezquinndesign** [comment]
Also happening on iOS builds using `@shakebugs/react-native-shake@17.0.0+`.

**Comment by almasios** [comment]
Any estimate when this will be fixed?

**Comment by stwonary** [comment]
I’m facing the same issue with the ShakeBugs Flutter SDK when uploading to TestFlight. The workaround you provided works for me as well. Thanks for sharing it!

**Comment by mstanic-shake** [comment]
Hi everyone, sorry for delayed response.

This should be fixed now in 17.0.2 version. Let us know if it works for you without workarounds now.

For RN/Flutter SDK, you can still use any 17.x version, just make sure to refetch dependencies and rebuild the app
(This will automatically pull latest native version).
